pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -72,6 +72,11 @@ type HyperConvergedFeatureGates struct {
 	// Support migration for VMs with host-model CPU mode
 	// +optional
 	WithHostModelCPU *bool `json:"withHostModelCPU,omitempty"`
+
+	// Enable HyperV strict host checking for HyperV enlightenments
+	// Defaults to true, even when HyperConvergedFeatureGates is empty
```

Comment:
I find it confusing. What do you think about setting this during reconciling? Somethink like this:

```golang
	if req.Instance.Spec.FeatureGates == nil {
		trueVal := true
		req.Instance.Spec.FeatureGates = &hcov1beta1.HyperConvergedFeatureGates{
			WithHostModelCPU: &trueVal,
		}
	}
```

Reply:
proper defaulting on read?
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting

Reply:
Yeah, I think this should be the way to go: https://book.kubebuilder.io/cronjob-tutorial/webhook-implementation.html